pkg/webui/console/containers/top-entities-dashboard-panel/top-entities-panel.styl

```diff
@@ -17,31 +17,30 @@
   display: flex
   flex-direction: column
 
-.entity
-  &-list
+  &-outer-table
+    overflow-y: auto
+    height: 23rem
+    width: 100%
+    table-layout: fixed
     scrollbar-width: none
+
     &::-webkit-scrollbar
       display: none
-    &-gradient
-      position:absolute
-      bottom: 0px
-      left: 0px
-      width: 100%
-      height: 98px
-      border-radius: 0px 0px 14px 14px
-      background: linear-gradient(180deg, rgba(255, 255, 255, .2) 0%, var(--c-bg-neutral-min) 64.58%)
-      pointer-events: none // Prevents the gradient from blocking the click event
-  &-body
-    height: 20rem
-  &-row
-    min-width: 100%
-    display: inline-table
-    border-bottom: 1px solid var(--c-border-neutral-extralight)
-    &.last-row
-      margin-bottom: 56px
-  &-cell
-    min-width: 100%
-    &-divided
-      width: 33%
-    &-small
-      width: 2.5rem
\ No newline at end of file
+
+    &-header
+      top: 0
+      position: sticky
+      background: var(--c-bg-neutral-min)
+      z-index: 1
+
+.scroll-gradient
```

Comment:
You should be able to use the new `<ScrollFader />` component to do this.

Reply:
I am having a hard time using <ScrollFader /> for several reasons:
1. The heights are too small and the scroll is not visible, so I tried to increase those but it breaks the layout.
2. The opacity simply does not update on scroll, if I put it outside the table:
```
<ScrollFader className={styles.scrollGradient} ref={listRef} light>
    <Table>
        <Table.Head className={styles.topEntitiesPanelOuterTableHeader}>{columns}</Table.Head>
        <Table.Body emptyMessage={m.empty}>{rows}</Table.Body>
     </Table>
</ScrollFader>
        
```
3. Putting the fader inside the body breaks the table:
```
    <Table>
        <Table.Head className={styles.topEntitiesPanelOuterTableHeader}>{columns}</Table.Head>
        <ScrollFader className={styles.scrollGradient} ref={listRef} light>
              <Table.Body emptyMessage={m.empty}>{rows}</Table.Body>
        </ScrollFader>
     </Table>
```

What is the `fadeHeight` meant for?
Am I missing something here?

Reply:
First implementation is the right one. Guess I'd need to put an eye on this myself. Can you push the commit?

Reply:
done

Reply:
I looked into this again and I found a solution. Not sure it's the best one thi